packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -265,84 +273,51 @@ class FrameFlameChart extends CoreElement {
     return maxRight;
   }
 
-  void _handleMouseMove(MouseEvent e) {
```

Comment:
This isn't needed as we can calculate this information right on a mouse wheel event which avoids possible bugs due to inconsistent values. I did this refactoring trying to track down while zoom based scrolling wasn't quite centered correctly. The main reason was the flameChartInset but this refactor makes the code a bit simpler and more robust.